Tools/clinic/clinic.py

```diff
@@ -2713,7 +2713,7 @@ class CConverter(metaclass=CConverterAutoRegister):
     # Or the magic value "unknown" if this value is a cannot be evaluated
     # at Argument-Clinic-preprocessing time (but is presumed to be valid
     # at runtime).
-    default: object = unspecified
+    default: object
```

Comment:
Looks like removing this default here breaks things, and I don't think removing _this_ one in particular buys us anything in terms of making it more mypy-friendly?

```suggestion
    default: object = unspecified
```

Reply:
True, there's no need to remove this one.